src/games/strategy/triplea/ResourceLoader.java

```diff
@@ -184,7 +184,7 @@ public URL getResource(final String inputPath) {
     String path = resourceLocationTracker.getMapPrefix() + inputPath;
     return getMatchingResources(path).stream().findFirst().orElse(
         getMatchingResources(inputPath).stream().findFirst().orElseGet(
-            ()->null));
+            () -> null));
```

Comment:
They must have happened while formatting. Sorry...
Pressing Ctrl+F+Shift makes things easier most of the time.